### ISSUE TITLE
Body validation fail when using TimestampLastEditedTimeFilter in DB querry.

### ISSUE THREAD

**Issue by adrianhrinko** [notion-dotnet/notion-sdk-net#365]
**The issue:**
When querying a database with the TimestampLastEditedTimeFilter, body validation fails as the Timestamp value should contain "last_edited_time" instead of "last_modified_time".

**To reproduce:**

```
var dateFilter = new DateFilter("When", onOrAfter: DateTime.Now);
var editedFilter = new LastEditedTimeFilter(onOrAfter: DateTime.Today);
var queryParams = new DatabasesQueryParameters { Filter = new CompoundFilter(and: new Filter[] {dateFilter, editedFilter}.ToList()) };
 var pages = await client.Databases.QueryAsync(databaseId, queryParams);
```

https://github.com/notion-dotnet/notion-sdk-net/blob/f5c40c6d4f958649ff216fcbf82de760f22efafc/Src/Notion.Client/Models/Filters/TimestampLastEditedTimeFilter.cs#L42

**Fix:**
Replace "last_modified_time" with "last_edited_time". 
It works for me.

**Comment by KoditkarVedant** [comment]
Do you think it has something to with the Notion-Version we use? I wonder if that field was renamed recently. Would you like to submit a PR?

**Comment by Dmitry-VF** [comment]
I ran into the same issue. If you are interested in a fix, please see my pull request: https://github.com/notion-dotnet/notion-sdk-net/pull/369

**Comment by KoditkarVedant** [comment]
@Dmitry-VF Thank you for providing the fix. I will try to take a look at this following weekend. I'm just not able to spend time on this project for the past few months.

**Comment by KoditkarVedant** [comment]
This has been released as part of [4.2.0](https://github.com/notion-dotnet/notion-sdk-net/releases/tag/4.2.0)
